core/services/keystore/models_test.go

```diff
@@ -55,6 +58,12 @@ func TestKeyRing_Encrypt_Decrypt(t *testing.T) {
 	require.Equal(t, originalKeyRing.OCR[ocr2.ID()].OnChainSigning.D, decryptedKeyRing.OCR[ocr2.ID()].OnChainSigning.D)
 	require.Equal(t, originalKeyRing.OCR[ocr2.ID()].OffChainSigning, decryptedKeyRing.OCR[ocr2.ID()].OffChainSigning)
 	require.Equal(t, originalKeyRing.OCR[ocr2.ID()].OffChainEncryption, decryptedKeyRing.OCR[ocr2.ID()].OffChainEncryption)
+	// compare ocr2 keys
```

Comment:
unrelated to this PR, but looks like ocr2 keys are missing from the logPubKeys call in models.go